src/components/Modals/AddClientModal.jsx

```diff
@@ -114,6 +118,7 @@ const AddClientModal = ({ showAddClientModal, setShowAddClientModal }) => {
     } finally {
       runNotification(
         `"${userObject.givenName} ${userObject.familyName}" added to client list`,
+        'error',
```

Comment:
should this be `success` instead? Adding the client was successful yes? It was still green for me even with this though.